packages/neon-core/src/wallet/core.ts

```diff
@@ -87,7 +95,12 @@ export function getPublicKeyFromPrivateKey(
 export const getVerificationScriptFromPublicKey = (
   publicKey: string
 ): string => {
-  return "21" + publicKey + "ac";
+  return (
+    num2hexstring(OpCode.PUSHBYTES33) +
+    publicKey +
+    num2hexstring(OpCode.SYSCALL) +
+    InteropService.NEO_CRYPTO_CHECKSIG

```

Comment:
This really reveals the differences in implementation. Both are supposed to be similar in nature but here the usage is really awkward with one requiring a num2hexstring while the other is native.

Reply:
Yes. Maybe I should change InteropSerivce to type number instead of HEX?

Reply:
I think it might be better if we change the enum OpCode to use string since we dont really do arithmetic on the opcodes anyway. And the name InteropServiceCode would be more suitable since InteropService sounds like an implementation not an enum.

Reply:
Good, I'll try to implement this.